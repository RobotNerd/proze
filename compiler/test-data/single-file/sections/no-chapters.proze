Title: My Document
Author: Jon Document

Section: A named section

This is the text of the first section.

---

This is the text of the next section.
