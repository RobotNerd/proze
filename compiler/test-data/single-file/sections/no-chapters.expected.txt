My Document
by Jon Document


A named section

This is the text of the first section.

---

This is the text of the next section.
